plugin/ipythonPlugins/src/dist/iruby/iruby.js

```diff
@@ -53,13 +53,22 @@ define(function(require, exports, module) {
           shellID = myPython.utils.uuid();
         }
 
-        var base = _.string.startsWith(serviceBase, "/") ? serviceBase : "/" + serviceBase;
-        bkHelper.httpGet("../beaker/rest/plugin-services/getIPythonPassword", {pluginId: PLUGIN_NAME})
+        bkHelper.httpGet(bkHelper.serverUrl("beaker/rest/plugin-services/getIPythonPassword"), {pluginId: PLUGIN_NAME})
         .success(function(result) {
-          bkHelper.httpPost(base + "/login?next=%2E", {password: result})
+          bkHelper.httpPost(bkHelper.serverUrl(serviceBase + "/login?next=%2E"), {password: result})
           .success(function(result) {
+            var baseurl = bkHelper.serverUrl(serviceBase);
```

Comment:
not really since these files are loaded only if the plugin is started for the specific notebook. Once we'll implement a proper inheritance model for language plugins this will come automatically.